library/system/src/modules/FileChanges.rb

```diff
@@ -213,16 +213,24 @@ def CheckFiles(files)
       ret == :ok
     end
 
+    # Files that are really new
+    #
+    # @param files [Array<String>] candidate files that may be new
+    # @return [Array<String>]
+    def new_created_files(files)
```

Comment:
It's for consistency with `CheckNewCreatedFiles` (the new method is extracting part of its logic).